frontend-react/src/__mocks__/DeliveriesMockServer.ts

```diff
@@ -1,7 +1,43 @@
 import { rest } from "msw";
 import { setupServer } from "msw/node";
 
-import { RSDelivery } from "../network/api/History/Reports";
+import { RSDelivery, RSFacility } from "../config/endpoints/deliveries";
+import config from "../config";
+
+const { RS_API_URL } = config;
+
+const makeFacilityFixture = (
+    identifier: number,
+    overrides?: Partial<RSFacility>
+): RSFacility => ({
+    facility: overrides?.facility || "Facility Fixture",
+    location: overrides?.location || "DeliveriesMockServer.ts",
+    CLIA: identifier.toString(),
+    positive: overrides?.positive || 0,
+    total: overrides?.total || 0,
+});
+
+export const makeDeliveryFixture = (
+    id: number,
+    overrides?: Partial<RSDelivery>
+): RSDelivery => ({
+    deliveryId: overrides?.deliveryId || 0,
+    batchReadyAt: overrides?.batchReadyAt || "",
+    expires: overrides?.expires || "",
+    receiver: overrides?.receiver || "",
+    reportId: id.toString() || "",
+    topic: overrides?.topic || "",
+    reportItemCount: overrides?.reportItemCount || 0,
+    fileName: overrides?.fileName || "",
+    fileType: overrides?.fileType || "",
+});
+export const makeDeliveryFixtureArray = (count: number) => {
```

Comment:
I don't necessarily _like_ this but its adapting what the old InterfaceClass code was doing [here](https://github.com/CDCgov/prime-reportstream/pull/6977/files#diff-d1d3596b606a876c0dc14b43083c7d86dfa03587addee8b90e30854621080a6fL26-L61)